contracts/ligo/src/tzip16-compat/fa2_multi_nft_faucet.mligo

```diff
@@ -20,15 +20,19 @@ let nft_faucet_main (param, storage : nft_faucet_entrypoints * nft_faucet_storag
      let ops, new_assets = mint_tokens (mp, storage.assets) in
      ops, { storage with assets = new_assets; }
 
-let sample_storage : nft_faucet_storage = {
-  assets = { ledger = (Big_map.empty : ledger);
+let sample_storage : nft_faucet_storage =
+  let contents_name = "" in
+  let contents_description = "" in
+  let contents_interfaces = "" in
+  let contents = Bytes.pack("{" ^ contents_name ^ "," ^ contents_description ^ "," ^ contents_interfaces ^ "}") in
```

Comment:
content should be well formed JSON.
See https://github.com/tqtezos/smart-contracts/blob/9796871b8a5ba7c85587397afff6a9ccf483005c/fractional/test/origination.ts#L23-L31 and https://github.com/tqtezos/smart-contracts/blob/9796871b8a5ba7c85587397afff6a9ccf483005c/fractional/test/origination.ts#L73 for example

Reply:
This was an error. I thought I had made the same change here as I did in another file. I'll update this shortly.